### DISCUSSION TITLE
Nextcloud container not starting after updating to Nextcloud AIO v7.7.1

### DISCUSSION THREAD

**Discussion by Mecallie** [nextcloud/all-in-one#3882] [Questions] [answered]
### Discussed in https://github.com/nextcloud/all-in-one/discussions/2923

I seem to be having the exact same issue as in the discussion above. After starting the upgrade the Apache, Nextcloud and Redis containers fail to start. I am running the AIO on an Unraid server.

The error I am getting in the logs of the Nextcloud container is:

```
now              
-------------------------------
 2023-12-06 09:38:46.280007+00
(1 row)

+ '[' -f /dev-dri-group-was-added ']'
++ find /dev -maxdepth 1 -mindepth 1 -name dri
+ '[' -n '' ']'
+ set +x
Installing imagemagick via apk...
Enabling Imagick...
WARNING: opening from cache https://dl-cdn.alpinelinux.org/alpine/v3.18/main: No such file or directory
WARNING: opening from cache https://dl-cdn.alpinelinux.org/alpine/v3.18/community: No such file or directory
Configuring Redis as session handler...
The last Nextcloud update failed.
Please restore from backup and try again!
If you do not have a backup in place, you can simply delete the update.failed file in the datadir which will allow the container to start again.
              now              
-------------------------------
 2023-12-06 09:39:10.516292+00
(1 row)

+ '[' -f /dev-dri-group-was-added ']'
++ find /dev -maxdepth 1 -mindepth 1 -name dri
+ '[' -n '' ']'
+ set +x
Configuring Redis as session handler...
The last Nextcloud update failed.
Please restore from backup and try again!
If you do not have a backup in place, you can simply delete the update.failed file in the datadir which will allow the container to start again.
              now              
-------------------------------
```

The problem is: I cannot find this update.failed file. Should this be in the nextcloud-aio-nextcloud container? If so: how do I remove the file, seeing that container is rebooting like crazy and I cannot get a console long enough to do anything?

If you need any more logs please let me know!

**Comment by szaimen** [comment, marked as answer]
> Please restore from backup and try again!

**Reply by Mecallie** [reply]
I've found the update.failed file: it was not actually in a docker container, but in the data dir of /mnt/users/nextcloud/ of my unraid server. I've deleted the file and the update went a-ok. I now have some issues with the permissions/owner of the data dir, but I usually get that fixed after some cursing.

You can close the discussion :)
